### PR TITLE
Add some basic instructions for create the customization yaml file

### PR DESCRIPTION
**Pull request by alvarorm22**
**Description of the change**

Add some basic instructions for create the customization yaml file

**Benefits**

**Possible drawbacks**

**Applicable issues**

  - fixes #

**Additional information**

### CODE REVIEW

**Review by aitorch — APPROVED** [review]
LGTM

File: customizations/README.md [hunk 1]
```diff
@@ -39,6 +39,41 @@ helm instal .. -f customizations.yaml
 
 You can also override values through the command-line to `helm`. Adding the argument: `--set key=value[,key=value]`
 
+### Creating the customizations yaml file
+
+Here you can find some basic instructions in order to create the config yaml file for your environment:
+
+- The configuration file `customizations.yaml` will be composed with keys and their value, please do not define the same key several times, because they will be overrided between them. Each key in the yaml file would have subkeys for differents configurations, so all of them should be inside the root key. Example:
```
Comment (jvillarf) [inline customizations/README.md:46]: `s/overrided/overridden/`

File: customizations/README.md [hunk 2]
```diff
@@ -39,6 +39,41 @@ helm instal .. -f customizations.yaml
...
+    ```yaml
+    mapsApi:
+      autoscaling:
+        enabled: true
+        minReplicas: 2
+        maxReplicas: 3
+        targetCPUUtilizationPercentage: 75
+  ```
+
+- Check the text values of the `customizations.yaml` keys, they have to be setted between quotes. Example:
```
Comment (jvillarf) [inline customizations/README.md:56]: `s/setted/set/`

File: customizations/README.md [hunk 3]
```diff
@@ -39,6 +39,41 @@ helm instal .. -f customizations.yaml
...
+
+  ```yaml
+  appConfigValues:
+    selfHostedDomain: "my.domain.com"
+  ```
+
+  Note that integers and booleans values are setted without quotes.
```
Comment (jvillarf) [inline customizations/README.md:63]: `s/setted/set/`

**Review by jvillarf — CHANGES_REQUESTED** [review]
Additionally, please as @ManuelLR suggested, let's consolidate these instructions with https://github.com/CartoDB/carto-selfhosted-helm/pull/155/files#diff-22be7b2163c32925f5228017d7eabef17cd467eca5b56e7923a0541bf129edb0R23-R39

File: customizations/README.md [hunk 4]
```diff
@@ -481,3 +482,38 @@ appSecrets:
 ## Advanced configuration
 
 If you need a more advanced configuration you can check the [full chart documentation](../chart/README.md) with all the available [parameters](../chart/README.md#parameters) or contact [support@carto.com](mailto:support@carto.com)
+
+## Tips for create the customization Yaml file
```
Comment (miguelangelmorenochacon) [inline customizations/README.md:486]: tips for _creating_

File: customizations/README.md [hunk 5]
```diff
@@ -481,3 +482,38 @@ appSecrets:
...
+
+Here you can find some basic instructions in order to create the config yaml file for your environment:
+
+- The configuration file `customizations.yaml` will be composed with keys and their value, please do not define the same key several times, because they will be overridden between them. Each key in the yaml file would have subkeys for differents configurations, so all of them should be inside the root key. Example:
```
Comment (miguelangelmorenochacon) [inline customizations/README.md:490]: composed _with_ keys

Comment (miguelangelmorenochacon) [reply customizations/README.md:490]: for _different_ configurations

**Review by miguelangelmorenochacon — APPROVED** [review]
🇺🇸

**Review by alvarorm22 — COMMENTED** [review]
:+1:

**Review by jvillarf — APPROVED** [review]

**Pull request merged**